lib/active_admin/resource/attributes.rb

```diff
@@ -0,0 +1,44 @@
+module ActiveAdmin
+
+  class Resource
+    module Attributes
+
+      def default_attributes
+        resource_class.columns.each_with_object({}) do |c, attrs|
+          unless reject_col?(c)
```

Comment:
@deivid-rodriguez , 

Again I'm not going to change default rendering attributes.
Only one functional change is  that columns ends with _id they are not always foreign keys,  that's one of the reasons why rails content_columns is not suitable, (see foo_id in tests). Also that's why it is not about duplication of AR functionallity. 

Switching CSV export to use columns is out of scope since it is changing behaviour too, columns include also foreign keys and STI columns. 

Columns ended with _count.  They are not regular columns, they are sytem columns to cache counters, they were designed for different things, changing this behaviour anyway is not context of this PR, as well as sti column. User still can ovverride with activeadmin DSL.


Dropping id_column in index_table is bad idea, it is helper method and not just rendering primary key.